test/ClangImporter/cxx-exceptions-executable.swift

```diff
@@ -0,0 +1,39 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-build-swift %s -I %S/Inputs/custom-modules/ -o %t/cxx_exceptions -Xfrontend -enable-cxx-interop
+// RUN: %target-codesign %t/cxx_exceptions
+// RUN: %target-run %t/cxx_exceptions
+
+import CxxExceptions
+import StdlibUnittest
+
+var CxxExceptionsTestSuite = TestSuite("CxxExceptions")
+
+// Uncaught C++ exceptions terminate the program.
+CxxExceptionsTestSuite.test("UncaughtException") {
+  expectCrashLater(withMessage: "terminate called")
+  throwException()
+}
+
+// Exceptions can be thrown and caught within C++ code if they don't cross any
+// interop boundaries.
```

Comment:
... furthermore, Swift can codegen throw and try-catch in a C++ inline functions successfully.

Reply:
Extended the comment.